frontend/lib/resolution/disambiguation.cpp

```diff
@@ -1708,6 +1504,10 @@ static int testArgMapping(const DisambiguationContext& dctx,
   // and value tuples.
   // TODO: not sure how to reproduce this code in Dyno
 
+  f1Type = QualifiedType(QualifiedType::Kind::IN, normalizeTupleTypeToValueTuple(dctx.context, f1Type.type()));
+  f2Type = QualifiedType(QualifiedType::Kind::IN, normalizeTupleTypeToValueTuple(dctx.context, f2Type.type()));
+  actualType = QualifiedType(QualifiedType::Kind::IN, normalizeTupleTypeToValueTuple(dctx.context, actualType.type()));
```

Comment:
Shouldn't this code be in some kind of conditional checking if we are working with a tuple?

Reply:
added a conditional on tuple type